fcrepo-http-api/src/test/java/org/fcrepo/http/api/FedoraLdpTest.java

```diff
@@ -1037,12 +1037,12 @@ public void testPatchBinaryDescription() throws Exception {
 
         when(resourceFactory.getResource(mockTransaction, binaryDescId))
                 .thenReturn(mockNonRdfSourceDescription);
-        testObj.updateSparql(toInputStream("xyz", UTF_8));
+        testObj.updateSparql(toInputStream("INSERT DATA { <> <http://some/predicate> \"xyz\" }", UTF_8));
         verify(updatePropertiesService).updateProperties(
                 eq(mockTransaction.getId()),
                 anyString(),
                 eq(binaryDescId),
-                eq("xyz")
+                anyString()
```

Comment:
This seems to be weakening the test. Can we check for some part of the string that should be expected?